### ISSUE TITLE
Enderpearl isnt working properly

### ISSUE THREAD

**Issue by mariagomezjs** [kernitus/BukkitOldCombatMechanics#562]
## Information
* Server Version: Spigot 1.12.2
* OldCombatMechanics version: 1.9.0
* Server Log File: none
* OldCombatMechanics config file: default

## Problem Description
Enderpearl isnt working properly with default configuration, users who use enderpearls, they are supposed to take damage (5 dmg / 2.5 hearts). I know that this config exist:
```
projectile-knockback:
  enabled: true
  worlds: []
  damage:
    snowball: 0.0001
    egg: 0.0001
    ender_pearl: 0.0001 <---
```
But if I modify this one, it will hurt to someone who is hit by that enderpearl aswell, this means both players will get the dmg, instead of the one that is throwing it.

### To Reproduce
1. Download the plugin
2. Put in the plugin folder, with default config
3. Try to use enderpearl in survival mode

**Comment by kernitus** [comment]
Can you please enable `debug` mode in the config and provide the damage outputs when an enderpearl is launched?

**Comment by mariagomezjs** [comment]
When I throw the enderpearl:
![when someone throw enderpearl](https://user-images.githubusercontent.com/92905476/141702723-1a20946e-2768-411f-8fd4-c195ebfbaf8b.png)

When someone is hit by an enderpearl:
![when someone hit you with enderpearl](https://user-images.githubusercontent.com/92905476/141702726-5df3f6c0-a37b-438f-be6b-6a30dc831dd6.png)

**Comment by kernitus** [comment]
Wait, that's for the wrong module - is there no output about the actual damage done and taken?

**Comment by mariagomezjs** [comment]
Nope, that's the only thing that I get when doing it.

**Comment by kernitus** [comment]
Fixed with 1df299b
